gradio/layouts.py

```diff
@@ -128,15 +132,19 @@ class Tabs(BlockContext):
 
     def __init__(
         self,
+        *,
         selected: Optional[int | str] = None,
+        visible: bool = True,
```

Comment:
I think we also need to modify the update method of Tabs class so that users can update the visibility in the demo